frontend/app/chat/[chatId]/components/ActionsBar/components/Feed/hooks/useFeed.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+import { FeedItemType } from "../types";
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const useFeed = () => {
```

Comment:
useKnowledgeToFeed to be more explicit about this being the area  displaying the knowledge to be fed